packages/core/src/components/Onboarding/index.tsx

```diff
@@ -19,6 +19,7 @@ import { PickPrimary } from './pages/PickPrimary'
 import { Referral } from './pages/Referral'
 import { ReviewPrimary } from './pages/ReviewPrimary'
 import { Welcome } from './pages/Welcome'
+var namer = require('color-namer')
```

Comment:
Yeah can we use es6 import syntax here?

Reply:
Thanks guys @alexdanilowicz @Pranav2612000. I will make those changes and push again.